src/main/kotlin/lotto/domain/LottoMachine.kt

```diff
@@ -1,10 +1,9 @@
-package lotto
+package lotto.domain
 
 class LottoMachine {
-    fun buy(money: Int): List<Lotto> {
-        require(money >= 1000) { ERROR_NOT_ENOUGH_MONEY }
+    fun buy(money: Money): List<Lotto> {
```

Comment:
아래 코드로 개선해 봅시다. (일급 컬렉션)
```suggestion
    fun buy(money: Money): Tickets {
```